src/stores/charts-store.ts

```diff
@@ -0,0 +1,22 @@
+import { types } from "mobx-state-tree";
+
+const Chart = types.model("Chart", {
+  type: types.union(types.literal("scatter"), types.literal("radial")),
+  data: types.array(types.array(types.number)),      // (x,y) or (deg,mag) tuples: [[x,y], [x,y], ...]
```

Comment:
At some point we might want to also include a date attribute. TBD

Reply:
Added in the next PR.